indexing/src/main/resources/org/allenai/common/indexing/indexing.conf

```diff
@@ -1,136 +1,88 @@
-org.allenai.common.indexing: {
+include "mappingHelpers.conf"
 
-  stoplist: {
+org.allenai.common.indexing {
+  stoplist {
     group: "org.allenai.nlp.resources"
     name: "smart_stopwords_2.txt"
     version: 1
   }
-
 }
 
-org.allenai.common.indexing.base: {
-  elasticSearch: {
+org.allenai.common.indexing.base {
+  elasticSearch {
     // Specify the right clustername. It usually defaults to `elasticsearch` but
     // you can verify by using the command:
     // curl -XGET 'http://localhost:9200/_cluster/health?pretty=true'.
-    // For Aristo Solvers, the hostIp should be changed to `aristo-es1.dev.ai2`
+    // For Aristo Solvers, the hostIp should be changed to `aristo-es.dev.ai2`
     // with clusterName `aristo-es`.
     clusterName: "elasticsearch"
     hostIp: "127.0.0.1"
     hostPort: 9300
```

Comment:
It's weird to me that we've got an explicit config setting for `hostPort: 9300` and reference an implicit port number `9200` a few lines later. Would it make sense to rename `hostPort` to something like `transportPortNumber`?

Reading through https://www.elastic.co/guide/en/elasticsearch/guide/current/_talking_to_elasticsearch.html it sounds like `9200` and `9300` are default values -- is `hostPort: 9300` needed?